cozy/model/settings.py

```diff
@@ -78,14 +77,12 @@ def _load_all_storage_locations(self):
             try:
                 self._storages.append(Storage(self._db, storage_db_obj.id))
             except InvalidPath:
-                log.error("Invalid path found in database, skipping: {}".format(storage_db_obj.path))
+                log.error(f"Invalid path found in database, skipping: {storage_db_obj.path}")
```

Comment:
Not sure if you were planning to use the printf-style formatting for logging as per [this comment](https://github.com/geigi/cozy/pull/825/files#r1427524245), but if you do then this one was not addressed.